README.md

```diff
@@ -1,2 +1,29 @@
-# Hello-World
-my first repository test
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="testlogo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World Project ReadMe</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+  - [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [👥 Author](#authos)
+
+
+# 📖 Hello World <a name="about-project"></a>
+
+**Hello World** is my first microverse project, which i start to understand the basics of GitHub.
+
+## 🛠 Built With HTML & CSS <a name="built-with"></a>
+
+### 👤 Author
+  
+**Nurbol Sultanov**
+
+- GitHub: [@everinurmind](https://github.com/everinurmind)
```

Comment:
- [ ] Kindly don't forget to implement a [MIT license](https://choosealicense.com/licenses/mit/) because software license tells others what they can and can't do with your source code, so it's essential to make an informed decision. You can also set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). :+1: